README.md

```diff
@@ -29,74 +29,74 @@ Platforms
 * `isBrowser`, boolean, *optional*, is the flag to tell if the module works in the browser. If not set will be computed based on the presence of the global `window` value,
 * `debug`, boolean, *optional*, when set will print some extra debug information.
 
-See the JSDoc markdown documentation for the returned `resin` object in [DOCUMENTATION.md](https://github.com/resin-io/resin-sdk/blob/master/DOCUMENTATION.md).
+See the JSDoc markdown documentation for the returned `balena` object in [DOCUMENTATION.md](https://github.com/balena-io/balena-sdk/blob/master/DOCUMENTATION.md).
 
 Support
 -------
 
-If you're having any problem, please [raise an issue](https://github.com/resin-io/resin-sdk/issues/new) on GitHub and the Resin.io team will be happy to help.
+If you're having any problem, please [raise an issue](https://github.com/balena-io/balena-sdk/issues/new) on GitHub and the Balena team will be happy to help.
 
 Tests
 -----
 
-In order to run the Resin SDK test suite, set the following environment variables from an account that exists and doesn't have a billing account code:  
+In order to run the Balena SDK test suite, set the following environment variables from an account that exists and doesn't have a billing account code:  
 **WARNING: This will delete all applications and public keys of the test user**
 
-- `RESINTEST_EMAIL`: The main account email
-- `RESINTEST_PASSWORD`: The main account password
-- `RESINTEST_USERNAME`: The main account username.
+- `TEST_EMAIL`: The main account email
+- `TEST_PASSWORD`: The main account password
+- `TEST_USERNAME`: The main account username.
 
 You also have to provide the following environment variables from an account that doesn't yet exists:
 
-- `RESINTEST_REGISTER_EMAIL`: The email of the account to register.
-- `RESINTEST_REGISTER_PASSWORD`: The password of the account to register.
-- `RESINTEST_REGISTER_USERNAME`: The username of the account to register.
+- `TEST_REGISTER_EMAIL`: The email of the account to register.
+- `TEST_REGISTER_PASSWORD`: The password of the account to register.
+- `TEST_REGISTER_USERNAME`: The username of the account to register.
 
 In order to test the billing methods for a paid account, you also have to configure the following environment variables:
 
-- `RESINTEST_PAID_EMAIL`: The email of the paid account.
-- `RESINTEST_PAID_PASSWORD`: The password of the account.
+- `TEST_PAID_EMAIL`: The email of the paid account.
+- `TEST_PAID_PASSWORD`: The password of the account.
 
 *Note: The paid user's `account billing code` should be set to `testdev` so that it's tested against the test plan.*
 
-You can also, optionally, set the `RESINTEST_API_URL` environment variable in order to run the tests using a different API backend (eg: `https://api.resinstaging.io`).
+You can also, optionally, set the `TEST_API_URL` environment variable in order to run the tests using a different API backend (eg: `https://api.balenastaging.io`).
```

Comment:
I think it is https://api.balena-staging.com , at least that is what the current dashboard is...